File: Linear Search/linear_search.py
# Python3 code to linearly search x in arr[].
# If x is present then return its location,
# otherwise return -1


def search(arr, n, x):

	for i in range(0, n):
		if (arr[i] == x):
			return i
	return -1


# Driver Code
arr = [2, 4, 6, 8, 10, 12, 14]
x = 10
n = len(arr)

# Function call
result = search(arr, n, x)
if(result == -1):
	print('Element is not present in array!')
else:
	print('Element is present at position', result+1)

File: Linear Search/linear_search.cpp
// C++ code to linearly search x in arr[]. If x
// is present then return its location, otherwise
// return -1

#include <iostream>
using namespace std;

int search(int arr[], int n, int x)
{
	int i;
	for (i = 0; i < n; i++)
		if (arr[i] == x)
			return i;
	return -1;
}

// Driver code
int main(void)
{
	int arr[] = { 2, 4, 6, 8, 10, 12, 14 };
	int x = 7;
	int n = sizeof(arr) / sizeof(arr[0]);

	// Function call
	int result = search(arr, n, x);
	(result == -1)
		? cout << "Element is not present in array!" << endl
		: cout << "Element is present at position " << result+1 << endl;
	return 0;
}

